index.html

```diff
@@ -601,9 +601,32 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 or local resources.</li>
 <li>The time immediately after the user agent finish establishing
 the connection to the server to retrieve the resource, otherwise.
-<p>The returned time MUST include the time interval to establish
-the transport connection, as well as other time intervals such as
-SSL handshake and SOCKS authentication.</p>
+
+<p>The returned time MUST include the time interval to establish the transport
```

Comment:
Should these also be part of the bullet point list? Otherwise, I think that it should go on the same paragraph as the previous sentence.

Reply:
Sure